modules/core/src/main/java/org/apache/synapse/unittest/TestingAgent.java

```diff
@@ -118,7 +118,7 @@
                         isArtifactDeployed = true;
                         deploymentStats.put(key, TYPE_PROXY);
                         testSuiteSummary.setDeploymentStatus(Constants.PASSED_KEY);
-                        log.info("Proxy artifact deployed successfully");
+                        log.info("Proxy main test artifact deployed successfully");
```

Comment:
Shouldn't this be something like 'Primary test Proxy artifact deployed successfully.' "I didn't understand what "Proxy main test artifact deployed successfully" at the beginning.

Reply:
Done. Changed the logs as suggested.